models/BaseORMService.cfc

```diff
@@ -643,23 +643,14 @@ component accessors="true"{
 	}
 
 	/**
-     * Checks if the given entityName and id exists in the database, this method does not load the entity into session
+     * Checks if the given entityName and id exists in the database
 	 *
 	 * @entityName The name of the entity
 	 * @id The id to lookup
 	 */
 	boolean function exists( required entityName, required any id ){
-		// Do it DLM style
-		var count = ORMExecuteQuery(
-			"select count( id ) from #arguments.entityName# where id = ?",
```

Comment:
What if you escape the entity name:

```
[#arguments.entityName#]
or
'#arguments.entityName#'
```

Reply:
>You can force Hibernate to quote an identifier in the generated SQL by enclosing the table or column name in backticks in the mapping document.

Reply:
If backticks work ( which is a MySQL convention ) that will fix the issue. I'll give it a shot.